<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultichannelTableWidget</class>
 <widget class="QWidget" name="MultichannelTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,2">
     <item>
      <widget class="QWidget" name="widget_display" native="true">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::PreventContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="Button_turn_upper">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>26</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_turn_low">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>26</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_turn_reset">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>26</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="Button_text_dis">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_text_nodis">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_text_reset">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>选择分析对象</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>隐藏无关通道</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>参考向量</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>旋转向量</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>谐波：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox">
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="specialValueText">
            <string/>
           </property>
           <property name="suffix">
            <string>次谐波</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
